mdp_plan_exec/package.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+<package>
+  <name>mdp_plan_exec</name>
+  <version>0.0.0</version>
+  <description>The mdp_plan_exec package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag --> 
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="b.lacerda@cs.bham.ac.uk">Bruno Lacerda</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>GPLv2</license>
```

Comment:
prism-robots is GPLv2, and I recall reading some stuff and thinking that this should be the same, and not MIT. Not sure about it though.